recipe/meta.yaml

```diff
@@ -1,12 +1,12 @@
-{% set version = "0.13.1-20180305" %}
+{% set version = "0.13.1" %}
```

Comment:
Are you sure this version is higher then the previous one?

Reply:
We only care about the `gcc7` version of it and that one will have only 1 version of the package. Also, I'm thinking about pulling the odd version from the `main` channel too but I won't do that now. I need to do some tests first.